WISE-Community/WISE
# Modified tests to use ChromeHeadless instead of PhantomJS.

Also renamed npm target names to be more precise. 

Test that the following works:
1. run ```npm test``` and make sure that wise5 tests pass
2. run ```npm run test-site``` and make sure that site tests pass
3. run ```npm run watch-all-wise5``` and make sure that tests run when you make a change to a *.es6 file
4. run ```npm run watch-all-site``` and make sure that tests run when you make a change to a *.ts file

Closes #1915
